test/new-e2e/tests/agent-metric-logs/log-agent/utils/file_tailing_utils.go

```diff
@@ -59,33 +76,33 @@ func appendLog(s *LinuxFakeintakeSuite, content string, recurrence int) {
 	}, 2*time.Minute, 10*time.Second)
 }
 
-// checkLogFile verifies the presence or absence of a log file path
-func checkLogFilePresence(s *LinuxFakeintakeSuite, logPath string) {
-	t := s.T()
-	osType := s.Env().VM.GetOSType()
+// CheckLogFile verifies the presence or absence of a log file path
+func CheckLogFilePresence(th TestHelper, logPath string) {
+	t := th.T()
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Mark all these as helpers, it helps test logs readability
```suggestion
	t := th.T()
	t.Helper()
```